text/1946-intra-rustdoc-links.md

```diff
@@ -317,6 +320,7 @@ with the wrong prefix that is in the same namespace.
 E.g., given an `struct Foo`, it may be possible to link to it using `[enum Foo]`,
 or, given a `mod bar`, it may be possible to link to that using `[struct bar]`.
 
+In case a type is in 
```

Comment:
Dangling line?

Reply:
my bad, was initially going to expand there and then decided to take a different path. removed.